app/src/main/java/com/epfl/drawyourpath/database/FirebaseDatabase.kt

```diff
@@ -121,6 +127,10 @@ class FirebaseDatabase(reference: DatabaseReference = Firebase.database.referenc
         return database.child(FirebaseKeys.TOURNAMENTS_ROOT)
     }
 
+    private fun tournamentsKeysRoot(): DatabaseReference {
+        return database.child(FirebaseKeys.TOURNAMENTS_IDS_ROOT)
+    }
```

Comment:
Maybe add some documentations to explain the difference between the function tournamentsRoot and tournamentsIdRoot()